string name
cwru_ariac/Dimension size
cwru_ariac/Grid grid
string frameName
geometry_msgs/PoseStamped basePose
cwru_ariac/Part[] contains
osrf_gear/Kit kitAssigned
osrf_gear/Kit kitCompleted
cwru_ariac/BoundBox bound
int8 state
float64 priority

# state
int8 INIT = 0
int8 READY = 1
int8 DELIVERING = 2
int8 RETURNING = 3